addons/pos_sale/views/sales_team_views.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="crm_team_view_form_inherit_pos_sale" model="ir.ui.view">
```

Comment:
Yeah right ! We discuss about changing that, but it's not decided yet. So keep it like this. Confusion in my head ....